### ISSUE TITLE
Unable to remove server

### ISSUE THREAD

**Issue by FeatherBoy** [HenriWahl/Nagstamon#470]
I have added server with asterisk in name and I am unable to remove it. As soon as I restart the program, I still see server in place. Also at configuration folder, there is still conf file:
C:\Users\username\.nagstamon\servers\

**Comment by HenriWahl** [comment]
Hi,
the asterisk seems to confuse the file access as it is a wildcard character on any OS - seems problem comes from there. Stupid handling of Nagstamon anyway... did you solve your problem meanwhile?

**Comment by FeatherBoy** [comment]
Hi Henri,
Yes it was exactly what I did right after it appeared and it fixed it. I just reported it here as this is not much user friendly behavior of nagstamon. 
FeatherBoy

**Comment by HenriWahl** [comment]
Yes, this is a little bit stupid behaviour. So I will keep the issue open as a reminder.

**Comment by HenriWahl** [comment]
The last testing releases use URL encoding for server config filenames - can you please test them to see if the problem gets solved?

**Comment by mbirth** [comment]
With the testing release 3.3-20191114 I wasn't able to remove the default entry "Monitor server". I had to manually delete the file `server_Monitor%20server.conf`.

**Comment by kr4bat** [comment]
This Bug re-appeared in version 3.16.2 with Win11 - i had to delete a Server entry in $HOME/.nagstamon
The Filename of the Server appeared with urlencodes characters, something like https://monsrv.domain.de/xyz

**Comment by HenriWahl** [comment]
@kr4bat if the building action worked there shall be a version 3.17-20250402 - please give it a try.

**Comment by HenriWahl** [comment]
Did it improve for anybody?
